README.md

```diff
@@ -1,11 +1,15 @@
-## What is oah-bes-vm 
+## What is oah-bes-vm?
 
 A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity.
 
 ## Who are the end users?
 
 RT & BT activist of BeS community.
 
+### Usage scenarios:
+1. **Security Analyst**: RT & BT activist of BeS community can use oah-bes-vm to work on a specific open-source security project.
+
+2. **BeSLab Manager**: oah-bes-vm helps BeSLab Manager & Security Analyst to analyse and do RT & BT activities of open-source security projects. After analysing multiple open-source security projects using multiple static analysis tools (like SonarQube, Codeql, SBOM etc) by BeSLab Manager, Security Analyst can use oah-bes-vm to setup an isolated RT/BT environment for each spesific open-source security project to do RT & BT activities.
 ## Features of oah-bes-vm?
 
 1. Offers bundles of BeS tools that would enable users to fast track their Red Teaming / Blue Teaming activities by significantly reducing effort spent on setting up respective environments.
```

Comment:
@pramit-d @asa1997 @sudhirverma does not look to the point to me.

Reply:
@panickervinod Updated readme file.